### ISSUE TITLE
Need maps for the Halloween Keys

### ISSUE THREAD

**Issue by eric-ycw** [ironbane/IronbaneServerLegacy#377]
Players are complaining about the keys, which have no use at all till now.

**Comment by ghost** [comment]
Yeah, we don't really have a purpose for them. Issue is, we'd be making areas ex-post-facto, after the event when no one can even get them anymore.

Unless, of course, we do embed the Halloween event somewhere in the world. I had suggested something like this before, but no luck. At least then we can make it a difficult trek to get to there. Then all that work put forth doesn't prevent players from experiencing it if they didn't get a chance.

Or, we could add a second way to get the keys. Perhaps make them incredibly expensive on an NPC somewhere. Maybe a guy that sells rare items from past events, with a store called "Bob Dack's Nick Knacks." XD Always seen wearing the latest event armor. :3

Just a thought.

**Comment by bsparks** [comment]
I think that @samjar already has plans or partial works on these zones. The keys were supposed to be one time event (until next year) type things. They give access to special zones for those lucky players.

I _do_ want to maybe add grouping, OR make it a portal similar to the pumpkin ring that as long as one person has the key, then they can take other ppl with them.

**Comment by ghost** [comment]
I like the grouping idea. I just think that in at least some way we should 
allow other player w\o keys to experience something, if only as a sorry 
that we haven't made anything for the keyholders yet.

Sent with AquaMail for Android
http://www.aqua-mail.com

**Comment by bsparks** [comment]
why do players who don't have keys _need_ to have something? that's the point of being around for the events. get exclusive bragging stuff.

**Comment by eric-ycw** [comment]
If we add the maps to the keys, the other players won't be able to enjoy the map, and I don't think many players have the keys. It would be very unfair.

**Comment by ghost** [comment]
Well, like Ben said, if we add grouping, then we can allow players with the keys to be able to ferry around the player w/o them.

And @Warspawn I don't think players need acceess, I just think it would be nice to give them some sort of access. Instead of creating content and shoveling it away, creating content that will be around all year, but difficult to get to until next season.

There aren't that many players who got keys in the first place, right? 'Sides devs. I could be wrong. There's no fun in an area that's VIP for like 10 people. Plus we don't have allot of content.

Maybe instead of the key areas, re-release the mansiom. Just take the pumpkins, put them in a cave far from town, and take out the rare Halloween-themed items and quests. This way we can keep obtaining player feedback to continue tweaking the area until a main release next year.

So we win, the dedicated players win, and we've kept keys rare but still allowed new player to experience the event if they really want to.

**Comment by bsparks** [comment]
hmm well if that is the case, then I think there should be some quests for the keys, possibly that lead into the haunted mansion (that in order to bring back I believe has to be redesigned a bit).
